.rat-excludes

```diff
@@ -114,6 +114,11 @@ odb.sln
 odb.vcxproj.*
 odb.rc
 # BSD licensed files
+ComJSON.h
+ComJSON.cpp
+ComJSONFuncs.cpp
+ComJSONStringInfo.h
+ComJSONStringInfo.cpp
```

Comment:
Not sure if these are needed here, with the copyright restored and the ASF statements added, but I did not try running RAT without to verify.
Would like a better comment here than BSD license - specify PostgresSQL.